src/components/Button/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import './Button.style.css'
+
+export default function Button(props) {
```

Comment:
I used props instead of destructuring because for attribute disabled I used conditional syntax with "?.". Is it ok? If not how I can make it different?